.packit.yaml

```diff
@@ -1,22 +1,23 @@
 specfile_path: ansible-bender.spec
 actions:
   create-archive:
-    - "python3 setup.py sdist --dist-dir ."
-    - "sh -c 'echo ansible-bender-$(python3 setup.py --version).tar.gz'"
+    - "python3 -m build"
+    - "sh -c 'echo dist/ansible-bender-$(python3 -m setuptools_scm).tar.gz'"
```

Comment:
Maybe go for `ls -1` instead of `echo` — this will make sure that the expected file got created.